src/GovUk.Education.ExploreEducationStatistics.Content.ViewModels/RedirectViewModels.cs

```diff
@@ -2,7 +2,8 @@ namespace GovUk.Education.ExploreEducationStatistics.Content.ViewModels;
 
 public record RedirectsViewModel(
     List<RedirectViewModel> Publications,
-    List<RedirectViewModel> Methodologies);
+    List<RedirectViewModel> Methodologies,
+    List<RedirectViewModel> Releases);
```

Comment:
I think this change has caused some build failures in other tests.

Reply:
Yep, you're absolutely right! I was meant to come back to this and totally forgot going into the weekend.